src/api/article.ts

```diff
@@ -0,0 +1,147 @@
+import { ArticlePostRequestType, FeedResponseType } from '@/types/article';
+
+import { BASE_HEADER, BASE_URL } from '@/constants/auth';
+
+const accessToken = JSON.parse(window.localStorage.getItem('user'))?.token;
```

Comment:
localStorage를 다룰때 유틸 함수로 빼서 대략 아래와 같이 사용해주면 좋을 것 같은데요.
JSON.parse와 같은 상세 구현은 숨기면 좋을 것 같고 'user'라는 key 없이 사용할 수 있도록 하면 휴먼에러를 방지하고 어떤 데이터를 담고 있는지 더욱 명확한 이름을 가질 수 있을 것 같아요. 추가로 getItem, JSON.parse 메소드에서 에러가 발생할수도 있을 것 같네요.

```suggestion
const accessToken = userLocalStorage.getItem().token;
```

Reply:
JSON.parse부분은 제가 급해서 차마 못 뻈습니다..ㅠㅠ
말씀하신 userLocalStorage로 만들어서 하는 부분을 생각해봤는데 이렇게 된다면 추후에 만들어지는 브라우저 스토리지마다 새로운 storage를 만들어야하는데 Class로 만들거나 아니면 차라리 하나의 함수 (get,set,remove)를 할 수 있고 react query를 훅화 할떄 쿼리키를 상수로 관리하는것처럼 사용하는건 어떨까요 ??
```tsx
const accessToken = LocalStorage.get(USER).token;
```